src/Services/DeadlineDateService.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+class DeadlineDateService
```

Comment:
I'd suggets renaming this now that it has a method that may not be directly related to a deadline - if you call this DateService instead I think it would make more sense.

There could be cases in the future where you need to reformat a date to UK style that is not a deadline